src/main/java/xyz/deseteral/lebkuchenfm/api/commands/text/TextCommandResponseDto.java

```diff
@@ -0,0 +1,34 @@
+package xyz.deseteral.lebkuchenfm.api.commands.text;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import xyz.deseteral.lebkuchenfm.domain.CommandProcessingResponse;
+import xyz.deseteral.lebkuchenfm.domain.NoSuchCommandProcessorException;
+import xyz.deseteral.lebkuchenfm.domain.TextIsNotACommandException;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+final class TextCommandResponseDto {
+    private final String response;
+
+    @JsonCreator
+    TextCommandResponseDto(@JsonProperty("response") String response) {
+        this.response = response;
+    }
+
+    TextCommandResponseDto(CommandProcessingResponse processingResponse) {
```

Comment:
A nie możesz przekazywać tego stringa wyżej? Jak to mawiał klasyk, lepsze kilka linii duplikacji niż tworzenie niepotrzebnych wiązań. Tak nie miałbyś zależności do wyjątków tutaj